js/main.js

```diff
@@ -21,23 +21,88 @@ function getData() {
 }
 getData();
 
-var likedThought = document.querySelector('#form-input');
 var text;
 var authorName;
 
-function handleClick(event) {
+function handleLike(event) {
   event.preventDefault();
 
   var savedQuote = {
     quote: text,
-    author: authorName
+    author: authorName,
+    entryId: data.nextEntryId
   };
 
   data.nextEntryId++;
-  likedThought.reset();
-  data.entries.unshift(likedThought);
-  ul.prepend(savedQuote);
+  // likedThought.reset();
+  data.quotations.unshift(savedQuote);
+  renderLiked(savedQuote);
+  viewSwap(savedQuote);
+}
 
+if ($button != null) {
+  $button.addEventListener('click', handleLike);
 }
 
-likedThought.addEventListener('click', handleClick);
+function renderLiked(entry) {
+  var card = document.createElement('p');
+  // card.className('card');
+  card.setAttribute('class', 'card');
+
+  var text = document.createElement('p');
+  text.textContent = entry.text;
+
+  var person = document.createElement('p');
+  person.textContent = entry.person;
+
+  return card;
+}
+
+var navLiked = document.querySelector('.nav-liked-thoughts');
+navLiked.addEventListener('click', viewSwap);
+
+function viewSwap(savedQuote) {
+  var parent = document.querySelector('.content');
+  parent.replaceChildren();
+
+  var entries;
+  if ('quote' in savedQuote) {
+    entries = [savedQuote];
+  } else {
+    entries = data.entries;
+  }
+
+  for (var k = 0; k < entries.length; k++) {
+    if (entries[k] !== null) {
+      if (k % 2 === 0) {
+        var row = document.createElement('div');
+        row.setAttribute('class', 'row');
+        parent.append(row);
+      }
+      var columnHalf = document.createElement('div');
+      columnHalf.setAttribute('class', 'column-half');
+      row.append(columnHalf);
+      var card = document.createElement('div');
+      card.setAttribute('class', 'card');
+      columnHalf.append(card);
+      var quoteList = document.createElement('ul');
+      quoteList.setAttribute('class', 'quote');
+      card.append(quoteList);
+      var quote = document.createElement('li');
+      quoteList.append(quote);
+      var author = document.createElement('li');
+      quoteList.append(author);
+
+      quote.textContent = entries[k].quote;
+      author.textContent = entries[k].author;
+
+      if ('quote' in savedQuote) {
+        var newButton = document.createElement('button');
+        newButton.setAttribute('class', 'button');
+        newButton.setAttribute('type', 'submit');
+        newButton.textContent = 'LIKE';
+        card.append(newButton);
+      }
+    }
```

Comment:
I would think this check of whether to make a SAVE button should be in the `renderLiked` function (Which should really be named `renderQuote`)